IdentityCore/src/network/session_delegate/MSIDURLSessionDelegate.m

```diff
@@ -25,4 +25,74 @@
 
 @implementation MSIDURLSessionDelegate
 
+#pragma mark - NSURLSessionDelegate
+
+- (void)URLSession:(NSURLSession *)session didReceiveChallenge:(NSURLAuthenticationChallenge *)challenge
```

Comment:
just curious, why do we need both this one and the URLSession:task:... methods?
Apple documentation says: `"If you do not implement this method, the session calls its delegate’s URLSession:task:didReceiveChallenge:completionHandler: method instead"`